unblob/cli.py

```diff
@@ -133,6 +133,13 @@ def __init__(
     show_default=True,
     help="Keep extracted chunks",
 )
+@click.option(
+    "-f",
+    "--force-override-extract-dir",
```

Comment:
Have you meant "overwrite"?

Reply:
Ohhhhh, you are right. I made it wrong everywhere.... thanks for bringing this up.